setup.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+import codecs
+from setuptools import (
+    setup,
+    find_packages,
+)
+from tor_core import __version__
+
+
+def long_description():
+    if not (os.path.isfile('README.md') and os.access('README.md', os.R_OK)):
+        return ''
+
+    with codecs.open('README.md', encoding='utf8') as f:
+        return f.read()
+
+
+setup(
+    name='tor_core',
+    version=__version__,
+    description='',
+    long_description=long_description(),
+    url='https://github.com/transcribersofreddit/tor_core',
+    author='Joe Kaufeld',
+    author_email='joe.kaufeld@gmail.com',
+    license='MIT',
+    classifiers=[
+        'Development Status :: 1 - Planning',
```

Comment:
I'm assuming you mean inclusion of the classifiers here. Since we're going with an initial version of `0.1.0` for this library, this descriptor makes sense again. Even if it didn't make sense, it doesn't matter since we're not publishing it to Pypi. No harm in leaving it as-is since it's currently accurate. 👍